### PR TITLE
Bump to v1.2.0

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

**Comment by basnijholt** [comment]
For some odd reason:
```
..../feedstock_root/build_artefacts/recipe_root_1485531881271/test_tmp/run_test.sh: line 3:   213 Segmentation fault      (core dumped) python test_tinyarray.py

```

File: appveyor.yml [hunk 1]
```diff
@@ -40,11 +32,11 @@ environment:
 
     - TARGET_ARCH: x86
       CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
 
     - TARGET_ARCH: x64
       CONDA_PY: 36
-      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
```
Comment (jbweston) [inline appveyor.yml]: I don't know what this means exactly, but is it correct that `CONDA_PY` is `36` but `CONDA_INSTALL_LOCN` contains `35`?

Comment (basnijholt) [inline appveyor.yml]: Yes, I beleive this is because Python 3.5 and 3.6 use the same Visual Studio version.

Comment (basnijholt) [reply appveyor.yml]: Yes, I believe this is because Python 3.5 and 3.6 use the same Visual Studio version.

**Pull request merged**